### DISCUSSION TITLE
Is it possible to export model properties to GLTF?

### DISCUSSION THREAD

**Discussion by thecatontheflat** [SteamDatabase/ValveResourceFormat#772] [General] [answered]
Hey 👋🏾 

I am trying to export some additional information which would help me to work with the exported model.

For example, when using the GUI, I can inspect such props as "targetname" and "classname", but apparently they aren't written to GLTF.

![image](https://github.com/ValveResourceFormat/ValveResourceFormat/assets/727581/a0e0d3b0-5d3e-4a7c-86d4-f7478f990dcd)

I assume this data is not exported currently.

Is this some functionality that you see being part of the project?
If so, could you please point to the relevant place in the code so I could try to extend it?

**Comment by xPaw** [comment, marked as answer]
It's all here: https://github.com/ValveResourceFormat/ValveResourceFormat/blob/master/ValveResourceFormat/IO/GltfModelExporter.cs

**Reply by thecatontheflat** [reply]
Thanks!
